src/mbed_tools/devices/device.py

```diff
@@ -60,6 +61,7 @@ def from_candidate(cls, candidate: CandidateDevice) -> "Device":
             mount_points=candidate.mount_points,
             mbed_board=mbed_board,
             mbed_enabled=mbed_enabled,
+            interface_version=device_file_info.interface_details.get("Version"),
```

Comment:
What tests that the device has `interface_version` set? I see tests for `interface_details` having it set, but not for the device.

Reply:
We test the device `interface_version` is set [here](https://github.com/ARMmbed/mbed-tools/blob/153dbed9d98c2ed517a1976b19ae4e178f9973b1/tests/devices/test_mbed_devices.py#L44)

Reply:
Thanks. I missed that.